wp-modules/pattern-data-handlers/pattern-data-handlers.php

```diff
@@ -374,68 +372,45 @@ function construct_pattern_php_file_contents( $pattern_data ) {
 
 /**
  * Scan all patterns in theme for images and other files, keep only ones actually being used.
- *
- * @param object $wp_filesystem The file system.
- * @param callable $copy_dir Copies a directory.
  */
-function tree_shake_theme_images( $wp_filesystem, $copy_dir ) {
+function tree_shake_theme_images() {
 	// Get the current patterns in the theme (not including templates and templates parts).
 	// Important note: we are not pulling in images from templates and parts because they are html files, and thus cannot reference a local image.
 	// Add the included Patterns for the current theme.
-	$theme_dir         = get_stylesheet_directory();
 	$patterns_in_theme = \PatternManager\PatternDataHandlers\get_theme_patterns();
-
-	$backedup_images_dir = $wp_filesystem->wp_content_dir() . 'temp-images/';
-	$images_dir          = $theme_dir . '/patterns/images/';
-
-	$wp_theme_url = get_stylesheet_directory_uri();
-	$images_url   = $wp_theme_url . '/patterns/images/';
-
-	if ( ! $wp_filesystem->exists( $backedup_images_dir ) ) {
-		$wp_filesystem->mkdir( $backedup_images_dir );
-	}
-
-	// Before we take any action, back up the current images directory.
-	call_user_func( $copy_dir, $images_dir, $backedup_images_dir );
```

Comment:
Just a refactoring of `tree_shake_theme_images()`, no intended functional change.

This is necessary because `copy_dir()` doesn't work in unit tests.

So we could only test `tree_shake_theme_images()` in isolation before, when we passed it a stub `copy_dir()`.

Now, we can test `tree_shake_theme_images()` as [part of](https://github.com/studiopress/pattern-manager/blob/da4f3075dcd30e66ab1d18bb1f5894c7db3df867/wp-modules/editor/model.php#L83) `save_pattern_to_file()`.